File: waktu_detik2.py
class Jam:
  def __init__(self, hh, mm, ss):
      self.hh = hh
      self.mm = mm
      self.ss = ss

print('Masukkan jam: ', end='')
hh = int(input())

print('Masukkan menit: ', end='')
mm = int(input())

print('Masukkan detik: ', end='')
ss = int(input())

j = Jam(hh, mm, ss)
totalDetik = (j.hh * 3600) + (j.mm * 60) + j.ss

print()
print('Total detik: ' + str(totalDetik))

File: WaktuDetik2.java
import java.util.Scanner;

public class WaktuDetik2 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        System.out.print("Masukkan jam: ");
        int hh = in.nextInt();

        System.out.print("Masukkan menit: ");
        int mm = in.nextInt();

        System.out.print("Masukkan detik: ");
        int ss = in.nextInt();

        Jam j = new Jam(hh, mm, ss);
        int totalDetik = (j.hh * 3600) + (j.mm * 60) + j.ss;

        System.out.println();
        System.out.println("Total detik: " + totalDetik);

        in.close();
    }
}
